examples/face_recognition_11/face_recognition_11/seed_test_run.py

```diff
@@ -0,0 +1,118 @@
+
+import pandas as pd
+from face_recognition_11.evaluator import evaluate_face_recognition_11
+from face_recognition_11.workflow import FMRConfiguration
+from face_recognition_11.workflow import Inference
+from face_recognition_11.workflow import Model
+from face_recognition_11.workflow import TestSample
+from face_recognition_11.workflow import TestSuite
+
+import kolena
+from kolena.workflow import test
+from kolena.workflow.annotation import BoundingBox
+from kolena.workflow.annotation import Keypoints
+
+BUCKET = "kolena-public-datasets"
+DATASET = "labeled-faces-in-the-wild"
+
+
+def seed_test_run(model_name: str, test_suite_names: List[str]) -> None:
+    df_results = pd.read_csv(f"s3://{BUCKET}/{DATASET}/predictions/predictions_{model_name}.sample.csv")
+
+    def infer(test_sample: TestSample) -> Inference:
+        sample_results = df_results[
+            (df_results["locator_a"] == test_sample.a.locator) & (df_results["locator_b"] == test_sample.b.locator)
+        ].iloc[0]
+
+        if sample_results["failure"]:  # failure to enroll
+            return Inference()
+
+        sample_results = defaultdict(None, sample_results)
+
+        return Inference(
+            left_bbox=BoundingBox(
+                (sample_results["a_min_x"], sample_results["a_min_y"]),
+                (sample_results["a_max_x"], sample_results["a_max_y"]),
+            ),
+            left_keypoints=Keypoints(
+                [
+                    (sample_results["a_right_eye_x"], sample_results["a_right_eye_y"]),
+                    (sample_results["a_left_eye_x"], sample_results["a_left_eye_y"]),
+                    (sample_results["a_nose_x"], sample_results["a_nose_y"]),
+                    (sample_results["a_mouth_right_x"], sample_results["a_mouth_right_y"]),
+                    (sample_results["a_mouth_left_x"], sample_results["a_mouth_left_y"]),
+                ],
+            ),
+            right_bbox=BoundingBox(
+                (sample_results["b_min_x"], sample_results["b_min_y"]),
+                (sample_results["b_max_x"], sample_results["b_max_y"]),
+            ),
+            right_keypoints=Keypoints(
+                [
+                    (sample_results["b_right_eye_x"], sample_results["b_right_eye_y"]),
+                    (sample_results["b_left_eye_x"], sample_results["b_left_eye_y"]),
+                    (sample_results["b_nose_x"], sample_results["b_nose_y"]),
+                    (sample_results["b_mouth_right_x"], sample_results["b_mouth_right_y"]),
+                    (sample_results["b_mouth_left_x"], sample_results["b_mouth_left_y"]),
+                ],
+            ),
+            similarity=sample_results["similarity"],
+        )
+
+    model = Model(f"{model_name} [{DATASET}]", infer=infer)
+    print(f"Model: {model}")
+
+    configurations = [
+        FMRConfiguration(false_match_rate=1e-1),
+        FMRConfiguration(false_match_rate=1e-2),
+        FMRConfiguration(false_match_rate=1e-3),
```

Comment:
I think 2 configs is sufficient for an example

Reply:
The legacy fr has these 6 different configs so I'm trying to follow that